### ISSUE TITLE
Curate bulk (non-MHC) proteomics in a separate index for detectability priors

### ISSUE THREAD

**Issue by iskandr** [pirl-unc/hitlist#67]
## Motivation

Right now hitlist indexes MHC-restricted MS evidence (ligandomes, elution assays) — and, separately, binding assays once #47 lands. A third category that belongs in the library is **bulk proteomics** on the same cell lines and tissues: whole-cell tryptic digests, deep proteome drafts, CPTAC tumor proteomes, etc.

The reason to curate this upstream rather than have each downstream tool re-scrape PRIDE/MassIVE/CPTAC: estimating **MS detectability bias by sequence content**. When a peptide is missing from an MHC-I ligandome, we'd like to know whether the parent protein is even expressed/detectable in that sample in general — otherwise "not presented" and "not detectable" collapse into the same observation.

## Proposal

Build a sibling `bulk_proteomics.parquet` alongside `observations.parquet` (MS, existing) and `binding_assays.parquet` (#47), keyed on **sample** (cell line, tissue, donor) rather than MHC allele.

### Schema sketch

| Column | Notes |
|--------|-------|
| `peptide` | tryptic (mostly) — caveat below |
| `protein_id`, `gene_name`, `gene_id` | from proteome sidecar, semicolon-joined for ambiguous mappings |
| `sample_id` | canonical cell line name (Cellosaurus CVCL_), tissue, or donor ID |
| `sample_kind` | `cell_line` / `tissue_tumor` / `tissue_normal` / `body_fluid` |
| `cell_line_name` | free-text (for cross-reference with `observations.cell_line_name`) |
| `tissue` | OBO-style tissue term when applicable |
| `disease` | free-text diagnosis where applicable |
| `protease` | `trypsin` / `lysc` / `aspn` / `chymotrypsin` / `multi` / `unknown` |
| `instrument` | Orbitrap Fusion, Q-Exactive, timsTOF, etc. |
| `intensity` | normalized abundance (log2 iBAQ or log2 LFQ) when available |
| `ms_level` | DDA vs DIA |
| `pmid`, `pxd_id` | PRIDE/MassIVE accession + paper |
| `source` | `ccle` / `cptac` / `kim2014` / `wilhelm2014` / `proteomicsdb` / `nci60` / ... |
| `is_contaminant` | cRAP/Krogan flagged |

### Candidate sources (first pass)

- **CCLE proteomics** — Nusinow et al. 2020 (Cell). ~375 cell lines, 11-plex TMT, tryptic. Direct overlap with many ligandome papers.
- **NCI-60** — Gholami et al. 2013. Tryptic, 60 lines.
- **Kim et al. 2014** and **Wilhelm et al. 2014** — draft human proteomes. Tissue-level, tryptic. Known to have contamination issues (Ezkurdia review); flag conservatively.
- **ProteomicsDB** — aggregated reference.
- **CPTAC** — tumor-matched bulk proteomes across BRCA, COAD, GBM, HNSCC, LUAD, OV, UCEC. Tissue-level, TMT, tryptic.
- **Geiger et al. 2012** — 11-cell-line super-SILAC tryptic reference.

### API sketch

\`\`\`python
from hitlist.bulk import load_bulk_proteomics

# Does the parent protein show up in any bulk MS run on this cell line?
load_bulk_proteomics(cell_line=\"A375\", gene_name=\"PRAME\")

# Peptide-level detectability lookup — is this exact tryptic peptide ever seen?
load_bulk_proteomics(peptide=\"LYVDSLFFL\")

# Aggregate to protein-level detection matrix
load_bulk_proteomics(sample_kind=\"cell_line\").groupby([\"gene_name\", \"sample_id\"])[\"intensity\"].max()
\`\`\`

## The tryptic caveat (important)

Bulk proteomics is overwhelmingly **tryptic** (cleaves C-term of K/R), while MHC-I ligandomes are **unrestricted** (proteasomal + aminopeptidase processing). This means:

- At the **peptide** level, you cannot directly compare \"this 9-mer was seen in MHC-I IP\" vs \"this 9-mer was seen in tryptic bulk\" — the tryptic peptide-space barely overlaps with the MHC-I peptide-space. The overlap that exists is mostly accidental (MHC-I peptide that happens to have K/R at position P1−1 and Pω).
- At the **protein** level, aggregation is clean: \"was any tryptic peptide from PRAME detected in A375 bulk MS?\" is a well-posed question and a useful prior for interpreting PRAME ligandome gaps.
- For **sequence-level detectability priors**, the right move is usually an in-silico model (DeepMSPeptide, Prosit detectability) trained on the aggregated bulk data — not direct peptide-level lookup.

**Recommendation:** ship the index with peptide-level rows (for people who want them and understand the caveat), but make the documented/blessed API protein-level aggregation. Add a `protease` column so users can subset to the (rare) non-tryptic bulk datasets if they want apples-to-apples peptide comparisons.

## Why separate file (not a kind-flag on observations)

Same reasoning as #47, plus:

1. **Different keys.** `observations.parquet` is peptide × allele. `bulk_proteomics.parquet` is peptide × sample. Joining them on either key drops most rows.
2. **Different scale.** CCLE + CPTAC + drafts is on the order of 10⁷–10⁸ peptide-sample rows. Mixing into observations would dominate the file.
3. **Different trust model.** Bulk MS detection is much noisier than targeted ligandome IP; conflating them in one table confuses per-row quality heuristics.

## Downstream impact

- **tsarina** — `tsarina hits --gene PRAME --allele HLA-A*24:02` could report \"PRAME protein is detected in bulk MS on cell line X at rank Y\" as a sanity check beside ligandome hits. Eventually: a `--require-bulk-detection` flag that only shows MHC-I hits whose parent protein is actually expressed in the sample's lineage.
- **Ligandome QC** — lets you flag studies where the MHC-I pulldown captures peptides from proteins that don't show up in bulk (contamination suspect) or conversely misses peptides from very abundant proteins (pulldown inefficiency).
- **Training data for detectability/retention models** — prebuilt index with normalized intensity is much friendlier than re-scraping PRIDE per project.

## Build/CLI surface

\`\`\`
hitlist data build                    # MS + binding (per #47)
hitlist data build --bulk             # add bulk proteomics (opt-in initially; big)
hitlist data build --bulk-sources ccle,cptac
\`\`\`

Default off until sources stabilize and size is measured, since this is a multi-GB index.

## Related

- hitlist#47 — binding-assay index (same separation-of-concerns pattern).
- hitlist#43 (closed, 1.6.0) — gene/protein annotation sidecar. The same sidecar applies here: peptide → gene_names/gene_ids/protein_ids is independent of whether the peptide came from MHC IP, in-vitro binding, or tryptic bulk.

**Comment by iskandr** [comment]
## Correction / clarification on the tryptic caveat

Rethinking the \"tryptic caveat\" section above — it was overstated and conflates two separate things. Splitting them out:

### Use 1: Per-sample expression prior  (requires cell-line matching)

\"Is PRAME even expressed in A375 at a level where we'd expect to see any of its MHC-I peptides?\"

Answer comes from protein-level aggregation of bulk MS on the matched cell line. Cell-line identity matters. Protease doesn't matter (any tryptic peptide detected → parent protein expressed).

### Use 2: Sequence-intrinsic detectability prior  (cell-line matching irrelevant)

\"Given its internal sequence, how detectable is peptide X by LC-MS/MS in general?\"

Training recipe — standard in the peptide-detectability literature (DeepMSPeptide, AP3, Prosit detectability head, DeepPep):

1. Take every protein that's **clearly expressed** in a bulk MS run (≥N high-intensity tryptic peptides detected).
2. In-silico digest it — enumerate every tryptic peptide the protease would produce.
3. Label each: detected in this run (y=1) vs. not detected (y=0).
4. Train P(detected | sequence features) — composition, hydrophobicity, charge, length, PTM propensity.

The protease is held constant across the whole training set, so the model isolates the **internal-sequence** detectability signal. That signal transfers to non-tryptic peptide spaces (including MHC-I ligands) because the underlying physics — ionization efficiency, fragmentation propensity, chromatographic retention — is sequence-intrinsic and doesn't care about the termini. There's a small termini-correction term, but the dominant variance is internal.

So:

- Peptide-level lookup (\"was this exact 9-mer seen in bulk MS?\") is ~useless for MHC-I peptides, as originally noted.
- **Training a detectability model on the curated bulk index and applying it to any peptide space** is exactly the use case, and works fine.

This strengthens the case for the index: it's not just an expression-prior lookup, it's also the training corpus for transferable detectability models. The schema already supports both uses without changes — for the detectability-training use, you pool across `sample_id` and condition on per-protein abundance rank. For the expression-prior use, you filter to the matched `sample_id`.

**Comment by iskandr** [comment]
**Scoping comment — first-ship-able increment:**

This is a large new index alongside \`observations.parquet\` and \`binding.parquet\`. The current build already distinguishes MS-elution vs binding-assay (#47 shipped); bulk proteomics is a third category.

Proposed first PR:
1. Define \`bulk_proteomics.parquet\` schema (per-sample, not per-MHC-allele): columns \`peptide\`, \`protein_id\`, \`gene_name\`, \`sample_id\` (cellosaurus/tissue), \`quantification\` (intensity/copies), \`source\` (cptac/proteomicsdb/massiveds/study-specific).
2. Wire an ingest path with ONE pilot dataset (e.g. CPTAC breast cancer draft proteome, or ProteomicsDB draft human proteome). Keep it to a single source.
3. Add \`load_bulk_proteomics()\` API mirroring \`load_observations()\` / \`load_binding()\`.

**Blocker for autonomous work:** The source data (CPTAC ~500GB, ProteomicsDB full dump, or per-study PRIDE deposits) must be downloaded and parsed. Each source has its own schema. ~1-2 week first PR per source.

Value: enables the \"not detectable vs not presented\" distinction at the parent-protein level — a common confounding factor in MHC ligandome analyses.

Not started.

**Comment by iskandr** [comment]
The separate bulk-proteomics index requested here has shipped in PRs #83, #84, #90, and #103. Rechecked current main: `hitlist.bulk_proteomics` exposes `load_bulk_proteomics()`, `load_bulk_peptides()`, and `load_bulk_sources()`; `builder.build_bulk_proteomics()` writes the separate parquet; the registry includes CCLE/Nusinow 2020 and Bekker-Jensen 2017.

The local built parquet contains 2,269,288 rows and retains evidence granularity, source/PMID, cell-line/sample identifiers, protein/peptide identifiers, normalized protein abundance, digestion enzyme, fractionation, enrichment, modifications, replicate counts, and acquisition metadata. This meets the first-shippable index/pilot/API increment specified in the discussion.

Closing this completed foundational issue. Remaining specific work stays open in #95 (peptide-level intensity), #96 (per-arm modifications), and #361 (detectability training set); this closure does not claim those enhancements or every suggested source are complete.
